include/odp/api/spec/shared_memory.h

```diff
@@ -96,6 +96,14 @@ extern "C" {
  */
 #define ODP_SHM_HW_ACCESS	0x20
 
+/**
+ * Don't use huge pages
+ *
+ * When set, this flag guarantees that the memory reserved by odp_shm_reserve()
+ * is not allocated from huge pages.
```

Comment:
Text could mention that this cannot be combined with ODP_SHM_HP

Reply:
That should be quite self-evident but an explanation won't hurt.

Reply:
Fixed in V2.